### PR TITLE
Add surfaceHub platform to HostClientType

### PR DESCRIPTION
**Pull request by kpyapali**
Add surfaceHub2 platform to HostClientType with value='surfaceHub'.
Use this value for SurfaceHub 2s specific scenarios/workflows.

### CODE REVIEW

**Review by msudankapoor — APPROVED** [review]

File: src/public/constants.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ export enum HostClientType {
   android = 'android',
   ios = 'ios',
   rigel = 'rigel',
+  surfaceHub2 = 'surfaceHub2',
```
Comment (sachingupta) [inline src/public/constants.ts:7]: as discussed please rename it to surfaceHub

Comment (ydogandjiev) [inline src/public/constants.ts:7]: As discussed in our sync please change this to the more generic `surfaceHub` instead of `surfaceHub2`.

**Review by ydogandjiev — CHANGES_REQUESTED** [review]

**Review by sachingupta — CHANGES_REQUESTED** [review]

File: src/public/constants.ts (hunk 1 above)
Comment (kpyapali) [reply src/public/constants.ts]: Changed.

Comment (kpyapali) [reply src/public/constants.ts]: changed.

**Review by sachingupta — APPROVED** [review]

**Review by ydogandjiev — APPROVED** [review]

**Pull request merged**